debian/appscale_install_functions.sh

```diff
@@ -573,3 +580,73 @@ upgradepip()
             ;;
     esac
 }
+
+buildmapreduce()
+{
+    # This function compiles the AppScale compatible MapReduce, Pipeline and GCS client libraries, builds the
+    # jars and places them under appscale/ext/java.
+
+    # Clone the AppScale repositories for the libaries.
+    git clone -b ${APPSCALE_GCS_BRANCH} ${APPSCALE_GCS_CLIENT} appscale-gcs-client
+    git clone -b ${APPSCALE_PL_BRANCH} ${APPSCALE_PIPELINE} appscale-pipeline
+    git clone -b ${APPSCALE_MR_BRANCH} ${APPSCALE_MAPREDUCE} appscale-mapreduce
+    echo -n "Building AppScale Google Cloud Storage Client jar ..."
+    if ! (cd appscale-gcs-client/java; mvn compile; cd target/classes; jar -cvf appscale-gcs-client-0.6.jar com/*); then
+        echo "GCS client compilation and jar creation failed!"
+        exit 1
+    fi
+
+    mkdir -p /root/appscale/ext/java/
```

Comment:
Do we have a constant for appscale HOME instead of /root/appscale here?
If so, please use it here.